qml/Stage/Stage.qml

```diff
@@ -967,34 +967,34 @@ FocusScope {
                         appDelegate.activate();
                     }
                     onStateChanged: {
-                        if (model.window.state === Mir.MinimizedState) {
+                        if (value == Mir.MinimizedState) {
```

Comment:
Same here: there's no reason why the old code wouldn't work, unless we have some serious problem.